flake.nix

```
@@ -85,6 +85,13 @@
               };
               taplo.enable = true;
               yamlfmt.enable = true;
+              check-summary-visualiser-html = {
+                enable = true;
+                name = "check-summary-visualiser-html";
+                entry = "${config.packages.check-summary-visualiser-html}/bin/check-summary-visualiser-html";
```

```suggestion
                entry = "nix run .#summary-visualiser-smoke-test`
```

Then it just works and you don't need the `check-summary-visualiser-html`writeShellScriptBin